src/actions/file-search.js

```diff
@@ -86,7 +86,7 @@ export function searchContents(query: string, editor: Object) {
     const matches = await getMatches(
```

Comment:
can we remove `      const _modifiers = modifiers.toJS();`?

Reply:
I think the tests threw when I did this last time, will retest to be sure.